cdap-ui/app/directives/widget-container/widget-factory.js

```diff
@@ -60,22 +60,24 @@ angular.module(PKG.name + '.commons')
         }
       },
       'csv': {
-        element: '<my-dsv></my-dsv>',
+        element: '<csv-widget></csv-widget>',
         attributes: {
-          'ng-model': 'model',
-          'delimiter': '{{::myconfig["widget-attributes"].delimiter}}',
-          'type': 'csv',
-          'config': 'myconfig'
-        }
+          'value': 'model',
+          'delimiter': 'myconfig["widget-attributes"].delimiter',
```

Comment:
nit: Should this be `,`? Do we allow users to provide a delimiter in csv widget too?

Reply:
for parity with the previous spec, we actually do allow that.

Reply:
Hhmm thats messed up. 
1. Our plugin presentation docs don't mention it ([docs](https://docs.cask.co/cdap/6.0.0/en/developer-manual/pipelines/developing-plugins/presentation-plugins.html#plugin-widgets))
2. I don't know what it means to say a comma-seperated-value widget and accept a delimiter 😐 

Anyways 👍

Reply:
this is just like in wrangler `parse-as-csv` but user still specify delimiter